src/index.js

```diff
@@ -0,0 +1,64 @@
+// Магазин
+
+import { Client } from './Client.js';
+import { Product } from './Product.js';
+import { Shop } from './Shop.js';
+
+let danylo = new Client(1, 'Danylo', 'Varlyhin');
+let yaroslav = new Client(2, 'Yaroslav', 'Isaenkov');
+
+let potato1 = new Product(1, 'Potato', 'Vegetables', 7);
+let potato2 = new Product(2, 'Potato', 'Vegetables', 7);
+let potato3 = new Product(3, 'Potato', 'Vegetables', 7);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Varlyle/my-first-project/pullRequest?prid=7306919)
```suggestion
let potato3 = new Product(3, "Potato", 'Vegetables', 7);
```